src/clusterfuzz/_internal/bot/fuzzers/syzkaller/runner.py

```diff
@@ -145,17 +198,77 @@ def repro(self, repro_timeout, repro_args):
     return engine.ReproduceResult(result.command, result.return_code,
                                   result.time_executed, result.output)
 
-  def fuzz(self,
-           fuzz_timeout,
-           additional_args,
-           unused_additional_args=None,
-           unused_extra_env=None):
+  def save_rawcover_output(self, pid: int):
+    """Find syzkaller port and write rawcover data to a file."""
+
+    port = find_syzkaller_port(pid)
+
+    # port not found
+    if not port:
+      logs.log_warn('Syzkaller port was not found')
+      return
+
+    rawcover = requests.get(f'http://localhost:{port}/rawcover').text
+
+    if not rawcover:
+      logs.log_warn('Syzkaller rawcover not yet loaded')
+      return
+
+    file_path = get_cover_file_path()
+    with open(file_path, 'w+') as f:
+      f.write(rawcover)
+      logs.log(f'Writing syzkaller rawcover to {file_path}')
+
+  def run_and_wait(self, syzkaller_args=None,
```

Comment:
can you make the signature:

```python
def run_and_wait(self, *args, timeout=None, **kwargs)
```

And then pass it to `self.run(*args, **kwargs)`

You also need to add a docstring here, something along the lines of "Overridden run_and_wait to ... describe why ...."

Reply:
fixed